enterprise-suite/gotests/testenv/testenv.go

```diff
@@ -87,6 +89,22 @@ func InitEnv() {
 		}
 	}
 
+	// Create an async goroutine to report when install is taking a while.
+	ticker := time.NewTicker(15 * time.Second)
```

Comment:
Not very important, but maybe this should be longer than 15 seconds? On my underpowered mac laptop minikube lbc install always takes a minute or so, so this would print repeated messages every time.